kubernetes/base/rbac.yaml

```diff
@@ -22,13 +22,10 @@ rules:
 - apiGroups: ["scheduling.k8s.io"]
   resources: ["priorityclasses"]
   verbs: ["get", "watch", "list"]
-- apiGroups: ["coordination.k8s.io"]
-  resources: ["leases"]
-  verbs: ["create"]
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
   resourceNames: ["descheduler"]
-  verbs: ["get", "patch", "delete"]
+  verbs: ["get", "patch", "delete","create", "update"]
```

Comment:
Is there a particular change/commit that resulting in need to add the verbs here? If so, it needs to be part of the same commit/PR.